python/ray/tune/schedulers/pbt.py

```diff
@@ -582,8 +583,20 @@ def _exploit(self, trial_executor: "trial_executor.TrialExecutor",
                 trial_executor.stop_trial(trial)
                 trial.set_experiment_tag(new_tag)
                 trial.set_config(new_config)
-                trial_executor.start_trial(
-                    trial, new_state.last_checkpoint, train=False)
+                # With placement groups, trial startup might not work
+                # first. This blocks until the trial has been started.
+                # We might want to introduce a timeout here or refactor
+                # how trials can be started/stopped with public methods.
+                timeout = time.monotonic() + 5  # hardcoded
```

Comment:
I'm not very happy about this, but generally it should work and it actually uses the same assumptions as we did before: Before we assumed that calling `stop_trial()` would free resources and `start_trial` would then use the same resources. Now we assume that `start_trial` eventually succeeds, as we assume that resources get cleared by `stop_trial` before. 

I can't think of a great way to deal with in another way this without refactoring PBT. Any thoughts?